Dev

voronin-ivan opened the pull request.

a-kvitko committed:
create index html

a-kvitko committed:
create folder css

a-kvitko committed:
create folder img, fonts. Market a header.

a-kvitko committed:
Merge branch 'dev' of https://github.com/alexkvitko/technomart into dev

a-kvitko committed:
create folder img, fonts. Market a header.

a-kvitko committed:
Corrections index html

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+    <header class="main-header">
+        <!-- Логотип, поиск, закладки, корзина, оформить заказ -->
+        <div class="header-line">
+            <section class="container">
```
Для контейнеров лучше использовать div. C новыми тегами HTML5 нужно быть предельно внимательными, вот хорошая шпаргалка - https://gaserge.ru/assets/resourcefiles/22/html5_tag-block.jpg

alexkvitko replied:
ты про это место?` <section class="container">` ?

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+            <section class="container">
+                <div class="main-logo">
+                    <img src="img/logo.png" alt="Техномарт" width="" height="">
+                </div>
```
Картинку тут не нужно оборачивать в div. Пусть сама картинка станет им! Допиши к картинке класс, впоследствии зададим ему display: block

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                    <img src="img/icon-search.png" alt="search" width="17" height="17">
+                    <span class="header-title">Поиск</span>
+                    <input type="text" name="input-search" id="input-search">
+                </div>
```
Вместо такого большого блока лучше сделать просто один input - "Поиск" будет у нас плейсхолдером, а картинку сделаем фоном.

alexkvitko replied:
Получается, что вместо всего дива останется только инпут? А как с классом для него быть? У инпутов вроде ID используется?
`<input type="text" name="input-search" id="header-search">`

voronin-ivan replied:
Да, только инпут. Класс можно добавлять **ко всем** тегам, даже `<hr>` (посмотри, что он делает). ID в современном вебе используется **только** для свзяки input + label и для якорных ссылок (link#4)

voronin-ivan reviewed (commented):
Нужно исправить

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                <div class="header-favorite">
+                    <img src="img/icon-favorite.png" alt="favorite" width="17" height="17">
+                    <span class="header-title">Закладки</span>
+                </div>
```
Слишком много элементов. Старайся руководствоваться правилом "чем меньше, тем лучше". Можно оформить эту кнопку одним элементом, вместо картинки - фон. + изменить на ссылку, по идее эта кнопочка ведет на страницу с закладками

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                    <img src="img/icon-basket.png" alt="basket" width="17" height="17">
+                    <span class="header-title">Корзина:</span>
+                    <span class="basket-number">0</span>
+                </div>
```
Комментарий выше

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                </div>
+                <div class="btn-buy">
+                    <a href="#" class="header-title">Оформить заказ</a>
+                </div>
```
Просто `<a href="#" class="btn-buy">`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                <div class="phone-block">
+                    <img src="img/icon-phone.png" alt="search" width="17" height="17">
+                    <span class="number-phone">+7 (812) 555-05-55</span>
+                </div>
```
Картинку лучше фоном, сам номер - ссылкой

alexkvitko replied:
> Картинку лучше фоном

а где это надо прописывать здесь же или в css?

voronin-ivan replied:
Картинку фоном - в css, ссылку - здесь, конечно. Прочитай по `<a> for tel`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                    <img src="img/icon-phone.png" alt="search" width="17" height="17">
+                    <span class="number-phone">+7 (812) 555-05-55</span>
+                </div>
+                <div>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</div>
```
А класс где?)

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+                <div class="login">
+                    <img src="img/icon-login.png" alt="login">
+                    <a href="#">Войти</a>
+                </div>
```
Картинку фоном, и тогда в элементе с классом user-block у тебя просто будет две ссылки.

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+        </div>
+        <!-- главное меню -->
+        <nav class="main-menu ">
+            <ul>
```
К ul очень желательно добавлять класс, чтобы li модифицировать через обращение к этому классу

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,94 @@
+
+        </div>
+    </main>
+=======
```
Убираем

a-kvitko committed:
error correction. start marks main block

a-kvitko committed:
edit: <a> for tel

a-kvitko committed:
marked popular goods block

a-kvitko committed:
minor fixes

a-kvitko committed:
marked brands block

a-kvitko committed:
Revert "marked brands block"

This reverts commit 94880774f0b6f8de4e003d9a9429fe4208972573.

a-kvitko committed:
Revert "Revert "marked brands block""

This reverts commit 368173b452ab9d91889c809a9d7f7d17bf01e1dd.

a-kvitko committed:
start marked service block

a-kvitko committed:
marked service block

a-kvitko committed:
marked information block

a-kvitko committed:
marked footer

Разметил футер и поправил кое-что в верхнем слайдере + проверил файл через валидатор

a-kvitko committed:
add label to input

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+        <title>Техномарт</title>
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+    </head>
```
Вплоть до body лучше не заморачиваться с форматированием, поэтому убираем пустые строки + `<head>` на том же уровне, что и `<html>`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+            <div class="container">
+                <!-- Логотип, поиск, закладки, корзина, оформить заказ -->
+                <div class="header-line">
+                    <div class="container">
```
У тебя два контейнера через строчку получаются) Должно быть так: main-header -> header-line -> container

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                <div class="header-line">
+                    <div class="container">
+                        <img class="main-logo" src="img/logo.png" alt="Техномарт" width="" height="">
+                        <label for="header-search">Поиск
```
Если input находится внутри label, то не надо присваивать ему id и писать for. слово "поиск" должно быть плейсхолдером к инпуту. Т.е., по сути, здесь спокойно можно обойтись без label

alexkvitko replied:
И как тогда должен выглядеть код для поиска?

voronin-ivan replied:
`<input type="text" class="header-search" placeholder="Поиск">`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                        </label>
+                        <a href="#" class="header-favorite">Закладки</a>
+                        <a href="#" class="header-basket">Корзина:</a>
+                        <a href="#" class="basket-number">0</a>
```
Зачем кол-во выводишь отдельно? Просто Корзина: 0

alexkvitko replied:
Думал что корзина будет всегда статичным, а количество меняться динамически. Нет?

voronin-ivan replied:
Верно думаешь, только на разметке это никак не отразится. Просто вместо 0 будет какой-то код, типа `<?php $count; ?>`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                    </div>
+                </div>
+                <!-- Слоган, контакты, авторизация -->
+                <div class="header-middleBlock">
```
Старайся избешать подобных названий. Стиль camelCase обычно в js используют, а не в разметке. Поэтому либо header-middle-block, либо просто header-middle (предпочтительнее)

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                    <div class="header-description">Интернет-магазин строительных материалов и&nbsp;инструментов для ремонта</div>
+                    <div class="header-contacts">
+                        <a href="tel:+7-812-555-05-55" class="phone-block">+7 (812) 555-05-55</a>
+                        <div class="adress">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</div>
```
Типограф

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                <div class="header-middleBlock">
+                    <div class="header-description">Интернет-магазин строительных материалов и&nbsp;инструментов для ремонта</div>
+                    <div class="header-contacts">
+                        <a href="tel:+7-812-555-05-55" class="phone-block">+7 (812) 555-05-55</a>
```
После tel: не должно быть форматирования вообще, т.е. 78125550555

alexkvitko replied:
а [+] форматированием считается?

alexkvitko replied:
а скобки у кода оператора тоже?

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                    </div>
+                </div>
+            </div>
+            <div class="container">
```
Лишний контейнер, смотри внимательно макет

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+            <div class="container">
+            <!-- главное меню -->
+                <nav class="main-menu">
+                    <ul class="main-menu-btn">
```
К ul почти всегда подходит название класса с list, т.е. main-menu-list

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+            <div class="container">
+            <!-- Блок с категориями товаров + слайдер -->
+                <section class="features">
+                    <div class="features-matireala">
```
matireala?

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                <section class="features">
+                    <div class="features-matireala">
+                        <span class="features-title">Материалы</span>
+                        <a class="features-btn" href="#">На любой вкус</a>
```
features -> feature, поскольку title и btn относятся к одному конкретному блоку. features-title подошло бы к заголовку всех этих блоков

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                    <div class="features-slider">
+                        <!-- Управление слейдером -->
+                        <img class="arrow-left" src="img/…" alt="arrow-left">
+                        <img class="arrow-right" src="img/…" alt="arrow-right">
```
Такие штуки лучше дивами с фоном сделать + к классам добавить префикс slider-

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                            <span class="slider-title">Перфораторы</span><br>
+                            <span class="slider-subtitle">Настоящие мужские игрушки</span>
+                            <a class="slider-btn" href="#">Открыть каталог</a>
+                        </div>
```
Обернуть в slider + замечения по slider->slide

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,295 @@
+                    <a class="pop-goods-item" href="#">
+                        <img src="img/pop-goods-item-1.jpg" alt="Перфоратор BOSCH BFG 9000" width="182" height="130">
+                        <span class="brand-title">Перфоратор BOSCH</span>
+                        <span class="brand-models">BFG 9000</span>
```
лишний класс, лучше название и модель обернуть в `<div class="brand-title">`, внутри которого будут два спана

a-kvitko committed:
added a catalog.html

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+            <section class="pop-brands">
+                <div class="pop-brands-line">
+                    <span>Популярные товары</span>
+                    <a class="pop-brands-lin" href="#">Все популярные производители:</a>
```
link

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <span>Популярные товары</span>
+                    <a class="pop-brands-lin" href="#">Все популярные производители:</a>
+                </div>
+                <div class="brands-logo-line1">
```
тут можно обойтись без класса

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <img class="brands-logo" src="img/pop-brands-logo-3.jpg" alt="DeWOLT" width="164" height="90">
+                    <img class="brands-logo" src="img/pop-brands-logo-4.jpg" alt="ИНТЕРСКОЛ" width="164" height="90">
+                </div>
+                <div class="brands-logo-line2">
```
как и тут

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+            </section>
+        </div>
+        <!-- Блок Сервис с табами -->
+        <div class="container">
```
Контейнер не тут должен быть

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+            <section class="service-block">
+                <div class="service-about">
+                    <span class="service-title">Сервисы</span>
+                    <p class="body-text">Хороший интернет-магазин отличается от плохого не только ценами! Мы стараемся изо всех сил, чтобы сделать ваши покупки приятными.</p>
```
Очень неудачное название класса. Почему не service-info, например. + типограф

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    </label>
+                </div>
+                <!-- Блоки с текстом и фоновой картинкой -->
+                <div class="tab-delivery-block">
```
Здесь лучше не использовать имя tab. Лучше что-то вроде service-delivery

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                <div class="tab-credit-block">
+                    <span class="tabs-title">Доставка</span>
+                    <p class="body-text">Залезть в долговую яму стало проще!<br>Кредитные консультанты придут вам на помощь.</p>
+                    <input type="submit" class="service-btn" value="Отправить заявку">
```
input type submit?) Ты, наверное, подразумевал `<button>`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+        <div class="container">
+            <section class="information-block">
+                <div class="company-info">
+                    <span class="title">Компания «ТЕХНОМАРТ»</span>
```
лучше что-то вроде company-title, title слишком обширно)

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <span class="title">Компания «ТЕХНОМАРТ»</span>
+                    <p>Настоящий мужской шоппинг начинается здесь! Огромный выбор товаров для ремонта и строительства не оставит равнодушными любителей сэкномить на гастарбайтерах.</p>
+                    <p>Мы можем доставить ваш товар в самые отдаленные точки России!<br>Техномарт работает со многими транспортными компаниями:</p>
+                    <div class="delivery-company">
```
Лишняя вложенность. Вместо дива сразу `<ul class="delivery-company>`

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                            <li>ЖелДорЭкспедиция</li>
+                        </ul>
+                    </div>
+                    <a class="btn-more" href="#">Подробнее о компании</a>
```
Опять слишком общее название. Лучше что-то вроде company-more-info

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    </div>
+                    <a class="btn-more" href="#">Подробнее о компании</a>
+                </div>
+                <div class="contact">
```
Блок на этом же уровне чуть выше ты назвал company-info. Почему здесь тогда просто contact? Лучше company-contacts

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <a class="btn-more" href="#">Подробнее о компании</a>
+                </div>
+                <div class="contact">
+                    <span class="title">Контакты</span>
```
Название класса

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <span class="title">Контакты</span>
+                    <p>Вы можете забрать товар сами, заехав в наш офис:</p>
+                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.5991043651586!2d30.320894616097018!3d59.938794181876254!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4696310fca5ba729%3A0xea9c53d4493c879f!2z0JHQvtC70YzRiNCw0Y8g0JrQvtC90Y7RiNC10L3QvdCw0Y8g0YPQuy4sIDE5LCDQodCw0L3QutGCLdCf0LXRgtC10YDQsdGD0YDQsywgMTkxMTg2!5e0!3m2!1sru!2sru!4v1504121127663" width="300" ...
+                    <a class="btn-writeToUs" href="#">Заблудились? Напишите нам!</a>
```
Опять camelCase используешь

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,294 @@
+                    <a class="phone-number" href="#">+7 (812) 555-05-55</a>
+                </div>
+                <ul class="site-map">
+                        <li>
```
отступы

a-kvitko committed:
minor fixes

a-kvitko committed:
started css

a-kvitko committed:
fixed style

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                <!-- главное меню -->
+                <nav class="main-menu">
+                    <ul class="main-menu-list">
+                        <li class="active">
```
active является дополнительным классом, забыл про основной - list-item. И я бы назвал не list-item, а menu-item, опять же из-за конкретики

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+        <!-- Логотип, поиск, закладки, корзина, оформить заказ -->
+        <div class="header-line">
+            <div class="container">
+                <div class="main-logo">
```
Лишний div, почему не просто img? А еще лучше <a> со ссылкой на главную страницу

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <div class="header-description">Интернет-магазин строительных материалов и&nbsp;инструментов для ремонта</div>
+                    <div class="header-contacts">
+                        <a href="tel:78125550555" class="phone-block">+7 (812) 555-05-55</a>
+                        <div class="adress">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</div>
```
Строчкой выше у тебя "phone-block", здесь тогда "address-block", так просто эстетичнее :)

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                        <div class="adress">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</div>
+                    </div>
+                    <div class="user-block">
+                        <a href="#">Войти</a>
```
А класса почему нет? Пусть будет "user-login", а ниже "user-registration", а не просто registration. Или button-login, как хочешь. Чем больше конкретики в названиях классов - тем лучше. Класс с наименованием registration лучше бы подошел к форме регистрации, например

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <div class="slider-arrow-left"></div>
+                    <div class="slider-arrow-right"></div>
+                    <ul class="slider-dots">
+                        <li class="slider-dot1">
```
Здесь лучше руководствоваться тем же принципом, что и в меню, где активная кнопка. Т.е. у двух точек одинаковый класс, например, slider-dot, и у активной какой-нибудь дополнительный, типа slider-dot-active

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                        <a class="slide-btn" href="#">Открыть каталог</a>
+                    </div>
+                </div>
+                <div class="features-sales">
```
все остальные блоки у тебя имеют класс, в названии которого feature в ед. числе. Внимательнее, это важно. Для слайдера ты сделал features-slider, и это правильно с точки зрения наименования, молодец

alexkvitko replied:
ok

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                </a>
+                <a class="pop-goods-item" href="#">
+                    <img src="img/pop-goods-item-1.jpg" alt="Перфоратор BOSCH BFG 2000" width="182" height="130">
+                    <img src="img/icon-new.png" alt="new" width="60" height="60">
```
Почему решил сделать это картинкой? Если докажешь (а здесь есть весомый аргумент в пользу твоего решения), то оставим так

alexkvitko replied:
ну я подумал, что это не часть интерфейса, потому и картинкой. По идее, это же карточки товара, а товар постоянно обновляется, и его надо как-то обновлять автоматически. я правда пока не знаю как это делается :) но думаю что использование именно картинок в этом случае будет правильным.

voronin-ivan replied:
Не убедил) Единственное явное преимущество - значение атрибута alt, куда можно написать, например, "Новый товар", что будет плюсом для seo (ведь нигде больше не указано текстом, что это новый товар). Я бы все-таки сделал дополнительный класс для самой карточки товара, но можно и так и оставить.

alexkvitko replied:
аааа, я не понял про какой место ты спрашивал :) теперь дошло. Речь о ярлычке NEW. Тут чисто интуитивно получилось, про сео я пока вообще не думал :)

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            <section class="pop-brands">
+                <div class="pop-brands-line">
+                    <span>Популярные товары</span>
+                    <a class="pop-brands-line" href="#">Все популярные производители:</a>
```
"популярные" - лишнее слово

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <span>Популярные товары</span>
+                    <a class="pop-brands-line" href="#">Все популярные производители:</a>
+                </div>
+                <div>
```
Можно одним дивом для всех картинок обойтись, либо вообще без него

alexkvitko replied:
здесь не понял

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            </section>
+        </div>
+        <!-- Блок Сервис с табами -->
+        <section class="service-block">
```
Контейнер потерялся

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+        </div>
+        <!-- Блок Сервис с табами -->
+        <section class="service-block">
+            <div class="service-about">
```
Можно без этого дива спокойно обойтись

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            <!-- Табы -->
+            <div class="three-tabs">
+                <label for="tab-delivery">Доставка
+                    <input type="radio" name="3tabs" id="tab-delivery" value="Доставка" checked="checked">
```
Используй короткую запись - просто checked

alexkvitko replied:
не знал, что так можно. fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                </label>
+            </div>
+            <!-- Блоки с текстом и фоновой картинкой -->
+            <div class="servie-delivery">
```
Внимательнее с названиями классов. servie, seervice (ниже) и подобное

alexkvitko replied:
упс… fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            </div>
+            <!-- Блоки с текстом и фоновой картинкой -->
+            <div class="servie-delivery">
+                <span class="tabs-title">Доставка</span>
```
К табам тут уже косвенное отношение, я бы подобрал другое название для класса

alexkvitko replied:
ну это же заголовки табов, почему нет?

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            <div class="service-credit">
+                <span class="tabs-title">Доставка</span>
+                <p class="seervice-info">Залезть в долговую яму стало проще!<br>Кредитные консультанты придут вам на помощь.</p>
+                <button type="submit" class="service-btn" value="Отправить заявку">
```
Очевидное задание type="submit" используется только для кнопок, которые отправляют форму. Эта кнопка будет ее лишь показывать, поэтому type лучше убрать.

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+            </div>
+        </section>
+        <!-- Блок с информацией и контактами с картой -->
+        <div class="container">
```
Лучше не container -> section, а section -> container

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+        <!-- Блок с информацией и контактами с картой -->
+        <div class="container">
+            <section class="information-block">
+                <div class="company-title">
```
company-title для такого большого блока? лучше company-info.

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <p>Настоящий мужской шоппинг начинается здесь! Огромный выбор товаров для ремонта и строительства не оставит равнодушными любителей сэкномить на гастарбайтерах.</p>
+                    <p>Мы можем доставить ваш товар в самые отдаленные точки России!<br>Техномарт работает со многими транспортными компаниями:</p>
+                    <ul class="delivery-company">
+                        <li>Деловые линии</li>
```
к лишкам классы было бы неплохо задать

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <span class="contacts-title">Контакты</span>
+                    <p>Вы можете забрать товар сами, заехав в наш офис:</p>
+                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.5991043651586!2d30.320894616097018!3d59.938794181876254!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4696310fca5ba729%3A0xea9c53d4493c879f!2z0JHQvtC70YzRiNCw0Y8g0JrQvtC90Y7RiNC10L3QvdCw0Y8g0YPQuy4sIDE5LCDQodCw0L3QutGCLdCf0LXRgtC10YDQsdGD0YDQsywgMTkxMTg2!5e0!3m2!1sru!2sru!4v1504121127663" width="300" ...
+                    <a class="btn-write-to-us" href="#">Заблудились? Напишите нам!</a>
```
Название класса из четырех слов уже слишком, старайся избегать. Тут можно просто btn-write-us

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <p>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
+                    <a href="tel:78125550555" class="phone-block">+7 (812) 555-05-55</a>
+                </div>
+                <ul class="site-map">
```
Если есть left-column, было бы неплохо иметь и right-column. Пусть этот ul и будет вложен в правую колонку. И это не совсем site-map, я бы сделал footer-menu

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+                    <a href="tel:78125550555" class="phone-block">+7 (812) 555-05-55</a>
+                </div>
+                <ul class="site-map">
+                    <li>
```
Классы к лишкам

alexkvitko replied:
fix

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+        </div>
+        <div class="footer-bottom-line">
+            <div class="container">
+                <p>© 2010–2016 Компания «Техномарт»<br>Все права защищены</p>
```
класс

voronin-ivan commented on index.html:
```diff
@@ -0,0 +1,299 @@
+        <div class="footer-bottom-line">
+            <div class="container">
+                <p>© 2010–2016 Компания «Техномарт»<br>Все права защищены</p>
+                <section class="social-block">
```
лучше div, на секцию не тянет

alexkvitko replied:
fix

a-kvitko committed:
fix errors in html

a-kvitko committed:
add .gitignore

a-kvitko committed:
fix errors html

a-kvitko committed:
Revert "fix errors html"

This reverts commit 8c9c24d190f08fd2c0bfa533db0e157661dfa985.

a-kvitko committed:
Revert "Revert "fix errors html""

This reverts commit 68046c4b9354fbaa7a3420b0a349f16326b68c4c.

a-kvitko committed:
reinstallation git

a-kvitko committed:
css for brands block

a-kvitko committed:
fix errors items block

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    background-color: #293547;
+}
+
+.header-line .container {
```
У контейнера должны быть единые свойства на всю страницу. Т.е. один раз написал .container {} и больше не трогаешь.

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    width: 226px;
+    padding: 11px 0 11px 46px;
+    border: none;
+    background-image: url("../img/icon-search.png");
```
лучше указывать абсолютный путь (почти всегда)

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    padding: 8px 34px 8px 26px;
+    font-size: 17px;
+    line-height: 26px;
+    background-image: url("../img/icon-favorite.png");
```
Это и два нижележащих свойства можно объединить в одно - background. Но это просто для инфы, в первое время лучше разделять, как ты и сделал.

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+.header-btn-buy {
+    display: inline-block;
+    vertical-align: middle;
+    background-color: #61A443;
```
Ты целых 114 строчек писал свойства в правильном, логичном порядке. Что тут случилось?) background-color в конец

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    font-size: 14px;
+    word-spacing: 4px;
+    color: #32425C;
+
```
лишняя строка

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+
+            /* MAIN */
+
+.container .features {
```
зачем через контейнер обращаешься?

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    text-transform: uppercase;
+    text-align: center;
+    color: #fff;
+    padding: 7px 12px;
```
padding после width

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+    color: #fff;
+}
+
+.slider-btn {
```
Всю навигацию слайдера пофиксить так, как мы с тобой обговаривали.

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+.goods {
+    position: relative;
+}
+
```
лишняя строка

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+
+.goods-img {
+    margin: 20px auto 17px;
+    display: block;
```
поменять местами свойства

voronin-ivan commented on css/style.css:
```diff
@@ -0,0 +1,638 @@
+}
+
+.service-info {
+    margin-bottom: 35px;
```
сначала width, потом margin

a-kvitko committed:
services block

a-kvitko committed:
service-block information-block edit

a-kvitko committed:
start footer block

a-kvitko committed:
hover and click effects

a-kvitko committed:
fix footer

a-kvitko committed:
fix footer

social block + pseudo elements

a-kvitko committed:
minor fixes

everywhere
